contracts/base/Escrow.sol

```diff
@@ -188,10 +188,12 @@ contract Escrow is EscrowInterface, TInitializable, BaseUpgradeable, BaseEscrowD
     */
     function claimTokens(address recipient) external {
         require(getLoan().status != TellerCommon.LoanStatus.Active, "LOAN_ACTIVE");
-        require(
-            recipient == getBorrower() || getLoan().liquidated,
-            "LOAN_NOT_LIQUIDATED"
-        );
+        if (getLoan().liquidated) {
+            require(recipient != getBorrower(), "LOAN_LIQUIDATED: recipient must not be the borrower");
```

Comment:
Please, review the error message standard (uppercase and _ ), and length <= 32 characters.